ios/Plugin/CapacitorCalendarPlugin.m

```diff
@@ -24,6 +24,7 @@
            CAP_PLUGIN_METHOD(deleteCalendar, CAPPluginReturnNone);
            CAP_PLUGIN_METHOD(getRemindersFromLists, CAPPluginReturnPromise);
            CAP_PLUGIN_METHOD(deleteRemindersById, CAPPluginReturnPromise);
+           CAP_PLUGIN_METHOD(deleteReminderById, CAPPluginReturnPromise);
```

Comment:
I dont see this method in the `definitions.ts`. Is this supposed to be `deleteEventById``

Reply:
This was probably meant to be `deleteEventById` yes, I changed it.